tools/ktx/command_compare.cpp

```diff
@@ -0,0 +1,2482 @@
+        <dt>\--ignore-supercomp</dt>
+        <dd>Ignore supercompression scheme in the file header. <br />
+            Note: use the --ignore-sgd option to also ignore the SGD section, if needed.</dd>
+        <dt>\--ignore-index all | level | none</dt>
+        <dd>Controls the comparison of index entries in the file headers. Possible options are: <br />
+            @b all - Ignore all index entries. <br />
+            @b level - Ignore level index entries only. <br />
+            @b none - Do not ignore any index entries. <br />
+            The default mode is @b none, meaning that all index entries will be compared.
+        </dd>
+        <dt>\--ignore-dfd all | all-except-color-space | unknown | extended | none</dt>
+        <dd>Controls the comparison of DFD blocks. Possible options are: <br />
+            @b all - Ignore all DFD blocks. <br />
+            @b all-except-color-space: Ignore all DFD blocks except the color space information
+                in the basic DFD block. <br />
+            @b unknown - Ignore any unrecognized DFD blocks. <br />
+            @b extended - Ignore all DFD blocks except the basic DFD block. <br />
+            @b none - Do not ignore any DFD blocks. <br />
+            The default mode is @b none, meaning that all DFD entries will be compared.
+        </dd>
+        <dt>\--ignore-bdfd-bytesplane</dt>
+        <dd>Ignore BDFD bytesPlane values. Useful when comparing supercompressed files with
+            non-supercompressed ones, as bytesPlane is set to zero for supercompressed files.</dd>
+        <dt>\--ignore-metadata all | &lt;keys&gt; | none</dt>
+        <dd>Controls the comparison of metadata (KVD) entries. Possible options are: <br />
+            @b all - Ignore all metadata entries. <br />
+            @b &lt;keys&gt; - Ignore the specified comma separated list of metadata keys. <br />
+            @b none - Do not ignore any metadata entries. <br />
+            The default mode is @b none, meaning that all metadata entries will be compared.
+        </dd>
+        <dt>\--ignore-sgd all | unknown | payload | none</dt>
+        <dd>Controls the comparison of the SGD section. Possible options are: <br />
+            @b all - Ignore the SGD section. <br />
+            @b unknown - Ignore any unrecognized SGD section. <br />
+            @b payload - Ignore any unrecognized SGD section and the payload of any known SGD section. <br />
+            @b none - Do not ignore the SGD section. <br />
+            The default mode is @b none, meaning that SGD sections will be always compared.
+            Note: --ignore-sgd payload can be used to compare BasisLZ SGD headers without
+            expecting an exact match for the individual SGD payload sections.
+        </dd>
+    </dl>
+    @snippet{doc} ktx/command.h command options_generic
+
+    Reports about individual texel block differences include the following information: <br />
```

Comment:
Reports about individual texel block differences, which are output if --content is set to @b image, include...

Reply:
I've used this phrasing, because the level, layer, and face is also output when --content is set to raw.